workbox.config.ts

```diff
@@ -0,0 +1,77 @@
+const SECOND = 1
+const MINUTE = 60 * SECOND
+const HOUR = 60 * MINUTE
+const DAY = 24 * HOUR
+const MONTH = 30 * DAY
+
+module.exports = {
+  // globPatterns: ["**/*.{html,png,jpg,jpeg,svg,gif,css,xml,js,json}"],
+
+  runtimeCaching: [
+    {
+      // Images
+      urlPattern: /.*\.(?:png|jpg|jpeg|svg|gif)$/,
+      handler: "CacheFirst",
+      options: {
+        cacheName: "images",
+        expiration: {
+          maxEntries: 50,
+          maxAgeSeconds: MONTH,
+        },
+      },
+    },
+    {
+      // Documents
+      urlPattern: /.*\.(?:html)$/,
+      handler: "NetworkFirst",
+      options: {
+        cacheName: "documents",
+        expiration: {
+          maxEntries: 70,
+          maxAgeSeconds: 14 * DAY,
+        },
+      },
+    },
+    {
+      // Styles
+      urlPattern: /.*\.(?:css)$/,
+      handler: "StaleWhileRevalidate",
+      options: {
+        cacheName: "styles",
+        expiration: {
+          maxEntries: 50,
+          maxAgeSeconds: 14 * DAY,
+        },
+      },
+    },
+    {
+      // Metadata
+      urlPattern: /.*\.(?:xml,json)$/,
+      handler: "StaleWhileRevalidate",
+      options: {
+        cacheName: "metadata",
+        expiration: {
+          maxEntries: 50,
+          maxAgeSeconds: MONTH,
+        },
+      },
+    },
+    {
+      // Scrips
+      urlPattern: /.*\.(?:js)$/,
+      handler: "CacheFirst",
+      options: {
+        cacheName: "scripts",
+        expiration: {
+          maxEntries: 50,
+          maxAgeSeconds: DAY,
+        },
+      },
+    },
+  ],
+
+  // Don't change that unless you know what you're doing
+  globDirectory: "public/",
+  swDest: "public/sw.js",
+  ignoreURLParametersMatching: [/^utm_/, /^fbclid$/],
```

Comment:
is there a way to ignore all params? iirc Quartz doesn't have any features that use query parameters for routing

Reply:
I removed the workbox config, not sure why it's still displayed here?

Reply:
try a `git rm workbox.config.ts`

Reply:
I opened a codespace and removed it there. Should be gone now :)